Gems/Atom/RHI/Code/Source/RHI/RayTracingAccelerationStructure.cpp

```diff
@@ -360,10 +363,14 @@ namespace AZ::RHI
                 return ResultCode::Success;
             });
 
-        if (const auto& name = m_tlasInstancesBuffer->GetName(); !name.IsEmpty())
+        if (m_tlasInstancesBuffer)
         {
-            m_tlasInstancesBuffer->SetName(name);
+            if (const auto& name = m_tlasInstancesBuffer->GetName(); !name.IsEmpty())
+            {
+                m_tlasInstancesBuffer->SetName(name);
+            }
```

Comment:
Same question as above, moving this to the lambda above this block

Reply:
Answered in the previous comment